mods/ts/rules/vehicles.yaml

```diff
@@ -837,4 +628,3 @@ STNK:
 	RenderVoxels:
 	WithVoxelBody:
 	-MustBeDestroyed:
-
```

Comment:
That's the danger of not looking at the full changes, I guess... nevermind, then.